static/js/formatter.js

```diff
@@ -0,0 +1,15 @@
+exports.format = function (msgs) {
+  var results = [];
+  for (const [id, msg] of Object.entries(msgs)) {
+    results.push({
+      inputs: {
+        localization_input_id: id,
+        source_text: msg.defaultMessage,
+        context: {
+          description: msg.description,
+        },
+      },
+    });
+  }
+  return results;
+};
```

Comment:
This is more FYI than anything @beets . If we really want, we can figure out how the proto<>json is and write the correct JSON. Or we can just write a simple parser that will take the default JSON and directly write a textproto.